EMIP/magicscrolls/EMIPms00469/EMIPms00469.xml

```diff
@@ -1,100 +1,137 @@
-         <change when="2022-09-02" who="CS">added include statements to incorporate traskribus output</change>
-         <change when="2022-09-23" who="RL">Corrected include code errors</change>
-         <change when="2022-09-26" who="RL">Removed text body code lines</change>
-</revisionDesc>
-   </teiHeader>
-      <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
-      href="facsimileEMIPms00469.xml">
-      <xi:fallback>
-      <!-- facsimile -->
-   </xi:fallback>
-</xi:include>
-      <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" 
-         href="transkribusTextEMIPms00469.xml">
-         <xi:fallback>
-            <!-- transkribus text -->
-         </xi:fallback>
-      </xi:include>   
-</TEI>
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="EMIPms00469">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title xml:id="t1">Tweed Magic Scroll 1</title>
+            <editor key="RL"/>
+            <editor key="CS"/>
+         </titleStmt>
+         <editionStmt>
+            <p>The initial version of this file was created from data kindly
+                                        provided by the Ethiopic Manuscript Imaging Project directed
+                                        by Steve Delamarter.</p>
+         </editionStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <pubPlace>Hamburg</pubPlace>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">This file is licensed under the Creative Commons
+                                                Attribution-ShareAlike 4.0.
+               </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0880WHU"></repository>
+                  <collection>Tweed collection</collection>
+                  <idno facs="EMIP/Scrolls/MS0469/" n="10">Tweed Magic Scroll 1</idno>
+                  <altIdentifier>
+                     <idno>EMIP MS 0469</idno>
+                  </altIdentifier>
+               </msIdentifier>
+
+               <msContents>
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                     <locus target="#facs_1_line_1656341948235_22"/>

```

Comment:
facs cannot be the target, `@facs` is a separate attribute, alongside `@target`, see https://betamasaheft.eu/Guidelines/?id=images Example 8

Reply:
@eu-genia I have added a target and facs statement, so I think this is now correct.

Reply:
Almost ) just move `#` from `facs `to `target`